core/src/test/scala/org/apache/spark/ContextCleanerSuite.scala

```diff
@@ -579,17 +579,20 @@ class CleanerTester(
 
   private def uncleanedResourcesToString = {
     s"""
-      |\tRDDs = ${toBeCleanedRDDIds.toSeq.sorted.mkString("[", ", ", "]")}
-      |\tShuffles = ${toBeCleanedShuffleIds.toSeq.sorted.mkString("[", ", ", "]")}
-      |\tBroadcasts = ${toBeCleanedBroadcstIds.toSeq.sorted.mkString("[", ", ", "]")}
+       |\tRDDs = ${toBeCleanedRDDIds.synchronized
```

Comment:
At this point it's getting hard to read this in the interpolated string. I'd just pull out the calls to `mkString` and save them as `val`s.